app/images_test.go

```diff
@@ -534,3 +538,136 @@ func TestGenerateImageSuccessfulWithTenant(t *testing.T) {
 	fs.AssertExpectations(t)
 	workflowsClient.AssertExpectations(t)
 }
+
+func TestGenerateConfigurationImage(t *testing.T) {
+	t.Parallel()
+	testCases := []struct {
+		Name string
+
+		DeviceType   string
+		DeploymentID string
+
+		StoreError error
+		Deployment *model.Deployment
+
+		Error error
+	}{{
+		Name: "ok",
+
+		DeviceType: "strawberryPlanck",
+		DeploymentID: uuid.NewSHA1(
+			uuid.NameSpaceOID,
+			[]byte("deployment"),
+		).String(),
+		Deployment: &model.Deployment{
+			Id: uuid.NewSHA1(
+				uuid.NameSpaceOID,
+				[]byte("deployment"),
+			).String(),
+			Type:          model.DeploymentTypeConfiguration,
+			Configuration: []byte("{\"foo\":\"bar\"}"),
```

Comment:
at the moment we are only supporting string values, I wonder how much of it should be tested here? perhaps a test that shows that we do not even car if it is json?